app/views/layouts/application.html.haml

```diff
@@ -7,9 +7,8 @@
     = stylesheet_link_tag    'application', media: 'all', data: {'turbolinks-track' => 'reload'}
     = javascript_include_tag 'application', data: {'turbolinks-track' => 'reload'}
   %body
-    #notifications
-      = alert 
-      = notice
-    = render 'partials/navbar'   
+    = alert 
+    = notice
+    -# = render 'partials/navbar'   
```

Comment:
Why do you remove this?

Reply:
We will open a CraftOverFlow on this matter, because we tried several ways. It goes in conflict with other buttons and it worked only when we removed the partial from application